pages/js-disabled.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { ReactElement } from "react";
+import { PublicFormRecord } from "@lib/types";
+
+const JSDisabled = ({
+  formProps,
+  lang,
+}: {
+  formProps: PublicFormRecord;
+  lang: string | undefined;
+}): React.ReactElement => {
+  const brandImg = formProps?.form; //formProps?.formConfig?.form;
+
+  return (
+    <>
+      <title>
+        {lang == "en"
+          ? "You need JavaScript to use this website."
+          : "JavaScript est requis pour consulter ce site Web."}
+      </title>
+      <main role="main">
+        <div className="block xs:mx-4 sm:mx-4 lg:mx-16 xl:mx-32 xxl:mx-48">
+          <div
+            className="flex justify-around md:h-auto"
+            style={{
+              flexDirection: lang == "en" ? "column" : "column-reverse",
+              height: "80vh",
+            }}
+          >
+            <div id="en">
+              <div className="flex justify-between items-center my-10">
+                <div className="xxs:w-flag-fold xs:w-flag-5s md:w-44 w-flag-desktop">
+                  {brandImg?.brand ? (
+                    <img src={brandImg?.brand?.logoEn} alt={brandImg?.brand?.logoTitleEn} />
```

Comment:
To avoid having eslint warnings you could add `/* eslint-disable @next/next/no-img-element */` at the top of the file with a comment explaining why you had to use `<img>` instead of `<Image>`.